terraso_backend/tests/graphql/test_sites.py

```diff
@@ -0,0 +1,46 @@
+# Copyright © 2021-2023 Technology Matters
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+import json
+
+import pytest
+
+from apps.soilproj.models import Site
+
+pytestmark = pytest.mark.django_db
+
+
+def test_site_creation(client_query):
```

Comment:
suggestion: if we are using a method of generating slugs that does not guarantee uniqueness, a test to make sure that duplicate slugs are rejected

Reply:
@shrouxm I switched to use the ID for slugs for now. I think we can work out the details of how we should generate slugs as it becomes more relevant.